Makefile

```diff
@@ -3,3 +3,7 @@ build:
 
 test:
 	go test -v $(shell go list ./... | grep -v /vendor/ | grep -v /test/)
+
+image: build
```

Comment:
As a Mac user I would appreciate to maybe create a `linux` target that executes something like:
`GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -i --ldflags="-s" go build -i github.com/pmorie/go-open-service-broker-skeleton/cmd/servicebroker`

And then call `linux` here instead of `build`.

Reply:
+1